fat-jar.html

```diff
@@ -0,0 +1,206 @@
+<h2>Creating a fat Jar</h2>
+
+<div class="alert alert-warning">
+<p>
```

Comment:
Adding `<p>` inside `<div>` with class "alert" results in adding space which doesn't look good.

Reply:
fixed